extensions/sonarr/package.json

```
@@ -0,0 +1,92 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "sonarr",
+  "title": "Sonarr",
+  "description": "View your Sonarr instance, upcoming shows and more",
```

In the future if you add more commands you can change this description
```suggestion
  "description": "View your Sonarr upcoming shows.",
```